hwdb.d/meson.build

```diff
@@ -38,7 +38,8 @@ hwdb_files_test = files(
         '70-software-radio.hwdb',
         '70-sound-card.hwdb',
         '70-touchpad.hwdb',
-        '80-ieee1394-unit-function.hwdb')
+        '80-ieee1394-unit-function.hwdb',
+        '82-net-auto-link-local.hwdb')
```

Comment:
I'm not sure if we should backport this. Effectively, the file was completely dead so it's as if it was added now with this commit. Should we add a new udev file in stable?

Reply:
The hwdb rules that we normally include in the backports are primarily just string descriptions. They do not provide "enablement" in general. We also have quirks about autosuspend and such… Those are not new features, just a better description of the existing hardware.

OTOH, adding this file enables something that is a new feature for users. We had a NEWS entry about ID_NET_AUTO_LINK_LOCAL_ONLY. But if it is advertised there as a new feature, we should not enable it here as part of a bugfix release. It's sad that the feature was merged partially ineffective, but when that happens, the solution is to wait for the next release.